content/v2.0/architecture/other-files.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Architecture: Overview"
+order: 20
+---
+
+## Anatomy Of A Project
+
+We have already examined `lib/` and `slices/`, but there are other parts of a newly generated project that deserve to be explained.
+
+```shell
+tree -L 1                                                                      
+.
+├── Brewfile
+├── Gemfile
+├── Gemfile.lock
+├── Guardfile
+├── Procfile.dev
+├── Procfile.support
+├── README.app.md
+├── README.md
+├── Rakefile
+├── babel.config.js
```

Comment:
According to the latest hanami2 template application, there is no `babel.config.js` file anymore https://github.com/hanami/hanami-2-application-template.

Seems there are other files now in the root dir, so probably worth a look and synchronizing with the application template.